anatomy/tracks/config-json.md

```diff
@@ -162,6 +163,20 @@ The "Recommended Order" of the Practice Exercises on the website corresponds wit
 }
 ```
 
+### Foregone exercises
+
+If a track knows that it doesn't want to implement an exercise defined in the [Problem Specifications repo](https://github.com/exercism/problem-specifications), the slug of that exercise can be added to the `exercises.foregone` key.
```

Comment:
Could you give an example of why a track would want to do this pls? Because I have no idea :)

"This is useful for..."

Reply:
Yeah sure. Will do that tomorrow.